Formula/sysstat.rb

```diff
@@ -0,0 +1,23 @@
+class Sysstat < Formula
+  desc "Performance monitoring tools for Linux"
+  homepage "https://github.com/sysstat/sysstat"
+  url "https://github.com/sysstat/sysstat/archive/v12.4.0.tar.gz"
+  sha256 "1962ed04832d798f5f7e787b9b4caa8b48fe935e854eef5528586b65f3cdd993"
+  license "GPL-2.0-or-later"
+  head "https://github.com/sysstat/sysstat.git"
+
+  depends_on :linux
+
+  def install
+    system "./configure",
+      "--disable-file-attr", # Fix install: cannot change ownership
```

Comment:
This indentation feels a bit weird. I'd suggest aligning with the first argument on line above.